apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -0,0 +1,87 @@
+
+The component API supports all standard HTML attribute props, while providing some additional behavior as described below.
+
+| Name        | Type                                   | Default | Required | Description                                                          |
+| :---------- | :------------------------------------- | :-----: | :------: | :------------------------------------------------------------------- |
+| `title`     | `string`                               |         |  `true`  | Sets the title of the video                                          |
+| `children`  | `React.ReactNode`, `React.ReactNode[]` |         |  `true`  | Takes `<source>` and `<track>` elements to present video information |
+| `poster`    | `string`                               |         | `false`  | Sets an image as the starting frame for the video element            |
+| `branding`  | `boolean`                              |         | `false`  | Displays GitHub branding next to video title                         |
+| `className` | `string`                               |         | `false`  | Sets a custom class                                                  |
+
+<h3>
+  VideoPlayer.Source <Label>Required</Label>
+</h3>
+
+`VideoPlayer.Source` provides a React alternative to the native HTML `<source />`.
+
+The component API supports all standard HTML attribute props, while providing some additional behavior as described below.
+
+| Name   | Type     | Default | Required | Description                           |
+| :----- | :------- | :-----: | :------: | :------------------------------------ |
+| `src`  | `string` |         |  `true`  | Sets the URL of the video content     |
+| `type` | `string` |         | `false`  | Sets media type for the video content |
+
+<h3>VideoPlayer.Track</h3>
+
+`VideoPlayer.Track` provides a React alternative to the native HTML `<track />`.
+
+The component API supports all standard HTML attribute props, while providing some additional behavior as described below.
+
+| Name      | Type                                                            | Default | Required | Description                                                                                                                              |
+| :-------- | :-------------------------------------------------------------- | :-----: | :------: | :--------------------------------------------------------------------------------------------------------------------------------------- |
+| `src`     | `string`                                                        |         |  `true`  | Sets the source of the VTT file containing captions                                                                                      |
+| `kind`    | `subtitles`, `captions`, `descriptions`, `chapters`, `metadata` |         | `false`  | Sets how the text track is meant to be used                                                                                              |
```

Comment:
These types should have quotes around them to indicate they are string types.

```suggestion
| `'kind'`    | `'subtitles'`, `'captions'`, `'descriptions'`, `'chapters'`, `'metadata'` |         | `undefined`  | Sets how the text track is meant to be used                                                                                              |
```

Reply:
Fixed in the following [SHA](https://github.com/primer/brand/pull/373/commits/ee6ae80b15875236b720a51258e132c66f39e856)